etc/inttypes-non-modular-header-workaround.diff

```diff
@@ -1,239 +0,0 @@
-diff --git a/Sources/CLibMongoC/bson/bson-iso8601.c b/Sources/CLibMongoC/bson/bson-iso8601.c
```

Comment:
@mbroadst This patch wasn't applying cleanly, so I tried just removing it and the driver seems to build and run fine on all our evergreen platforms. Do you think we're okay just leaving it off?

Reply:
hmm, I guess so! I can't remember what motivated the change originally, maybe it was something on Travis